sos/plugins/__init__.py

```diff
@@ -180,14 +180,17 @@ def is_installed(self, package_name):
         '''Is the package $package_name installed?'''
         return self.policy().pkg_by_name(package_name) is not None
 
-    def do_cmd_output_sub(self, cmd, regexp, subst):
+    def do_cmd_output_sub(self, cmd, regexp, subst, dotall=False,
+                          ignore_case=False):
         '''Apply a regexp substitution to command output archived by sosreport.
         cmd is the command name from which output is collected (i.e. excluding
         parameters). The regexp can be a string or a compiled re object. The
         substitution string, subst, is a string that replaces each occurrence
-        of regexp in each file collected from cmd. Internally 'cmd' is treated
-        as a glob with a leading and trailing '*' and each matching file from
-        the current module's command list is subjected to the replacement.
+        of regexp in each file collected from cmd. Regexp match is case
+        sensitive unless ignore_case is True. The dotall flag causes the
+        re.DOTALL flag to be used. Internally 'cmd' is treated as a glob with a
```

Comment:
Yeah - see my suggestion for a `do_cmd_output_delete()` method further down - I try not to say "NO!" without at least coming up with some alternative for discussion :-)

The name is horrible (feel free to suggest something better) but I think this is a better interface for this kind of "CUT HERE" method.